MovieKuno/Tab Pages/Home Views/HomeView.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  HomeView.swift
+//  MovieKuno
+//
+//  Created by Leah Joy Ylaya on 1/20/21.
+//
+
+import Foundation
+import SwiftUI
+
+struct HomeView: View {
+    @State private var movies: Movies = [
+        Movie(id: 1,
+              title: "Harry Potter and the Philosopher Stone",
+              posterURL: "",
+              voteAverage: "4.5",
+              releaseDate: "21 November 200",
+              genres: [1, 2, 3]),
+        Movie(id: 2, title: "Despicable Me 3",
+              posterURL: "",
+              voteAverage: "6.3",
+              releaseDate: "14 June 2017",
+              genres: [1, 2, 3]),
+        Movie(id: 3, title: "Monsters University",
+              posterURL: "",
+              voteAverage: "4.5",
+              releaseDate: "26 June 2013",
+              genres: [1, 2, 3])
+    ]
```

Comment:
Maybe we should move this placeholder data in its own Model class. We can refactor that class later on to work with actual API responses.